packages/idafree.vm/idafree.vm.nuspec

```
@@ -2,12 +2,13 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
   <metadata>
     <id>idafree.vm</id>
-    <version>8.4.0.20250219</version>
+    <version>8.4.0.20250430</version>
     <authors>Hex-Rays</authors>
-    <description>Free version of IDA Pro, a powerful Interactive DisAssembler and debugger.</description>
+    <description>Idafree is the free version of IDA Pro, a powerful Interactive DisAssembler and debugger.</description>
```

The tool name is: `IDA Free`. Please check the tool names in the official tool web page.

```suggestion
    <description>Idafree is the free version of IDA Pro, a powerful Interactive DisAssembler and debugger.</description>